src/cli/cli_coap_secure.cpp

```diff
@@ -393,6 +596,19 @@ otError CoapSecure::ProcessRequest(Arg aArgs[], otCoapCode aCoapCode)
     return error;
 }
 
+/**
+ * @cli coaps connect
+ * @code
+ * coaps connect fdde:ad00:beef:0:9903:14b:27e0:5744
+ * Done
+ * coaps connected
```

Comment:
Is this correct where it appears after Done?

(Comparing to coaps disconnect where the example has it before Done)

Reply:
@Vyrastas - will await reply from Abtin on this one, thanks Jeff

Reply:
@abtink  Thank you